include/GcInfo/Target.h

```diff
@@ -17,10 +17,19 @@
 #define GCINFO_TARGET_H
 
 #include "global.h"
+#include "corinfo.h"
 
 #if (defined(_TARGET_X86_) || defined(_TARGET_X64_) || defined(_TARGET_AMD64_))
 
-// Define DWARF encodings for registers
+// Identify the frame-pointer register number
```

Comment:
It would be good to have a ICorTargetInfo, containing target specific definitions such as registers etc., instead of picking it up from the ICorDebugInfo.